notebook/notebook/notebook.py

```diff
@@ -456,7 +460,7 @@ def notebook_status_from_pod(pod):
 
 
 @routes.get('/auth/{requested_notebook_token}')
-@web_authenticated_users_only()
+@web_authenticated_users_only(redirect=False)
```

Comment:
At some point I think it would be nice to redirect with a message.

Reply:
This is an internal endpoint, it is not possible to redirect here.  Only nginx can redirect based on the auth return code, which I'm now doing.

Reply:
Also, /auth can only return 2xx, 401 and 403.  All other error codes are treated as 500 by nginx in auth_request.

Reply:
Just, one more clarification: we do redirect with a nice error message.  nginx routes 401 and 403 to @error which redirects to /error which sets a message and redirects to /notebook.  Testing this is also in the test playbook.